### PR TITLE
Add GitHub description

### PR DESCRIPTION
**Pull request by gracjankn**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -6384,6 +6396,16 @@
         ]
     },
 
+    {
+        "name": "Revolut",
+        "url": "https://www.revolut.com/help/profile/i-want-to-close-my-revolut-account",
+        "difficulty": "easy",
+        "notes": "Open the Revolut app on your phone, go to the Settings tab, press 'Close account' button and follow the instructions.",
```
Comment (Nightfirecat) [inline _data/sites.json]: This should mention that they continue to store data for 6 years to comply with regulatory requirements.

Comment (gracjankn) [reply _data/sites.json]: Something went wrong. The commits for Revolut and GOG were already merged in a pull request I made a month or two ago. Only GitHub commit should be included in this pull request, but I don’t know how to fix that. I did a GitHub commit on a separate branch on my fork and I not longer have the old branch for the rest of the changes, but they still showed up.

Comment (Nightfirecat) [reply _data/sites.json]: You should rebase on the upstream master then

Comment (gracjankn) [reply _data/sites.json]: > You should rebase on the upstream master then

Ok, I’ll do it tomorrow.

**Comment by tupaschoal** [comment]
We can help you with rebasing if you're having problems. Or I can pull your changes into a PR. Was that why you closed?

**Comment by gracjankn** [comment]
Yes, that's the reason. I kinda tried to do something, but it still says master is 2 commits ahead and this branch is 3 commits ahead. I don't really know how I should fix it, because it is a first time I need to. My personal use of git is so simple, I almost solely use branches when adding things to other projects to projects like this.

**Comment by tupaschoal** [comment]
There's an "easy" way out, whenever you do a change and it gets merged, you delete your fork and let Github fork it again the next time.

If you're not willing to do this, you can always open issues to us describing what has changed or what is new and we can do the PR to merge it :)

**Comment by PyroSamurai** [comment]
Assuming he didn't do his edits on Github, he should rename his current repo on his PC, delete his fork on github, refork, reclone, then c/p his changes into the new fork's master, push to fork, make PR.

**Comment by tupaschoal** [comment]
Sure, I assumed it was being fully done on GitHub.

**Pull request closed without merging**